launch/tier4_perception_launch/launch/object_recognition/detection/detection.launch.xml

```diff
@@ -31,8 +31,6 @@
 
   <!-- Radar parameters -->
   <arg name="input/radar" default="/sensing/radar/detected_objects"/>
-  <arg name="radar_lanelet_filtering_range_param" default="$(find-pkg-share detected_object_validation)/config/object_lanelet_filter.param.yaml"/>
-  <arg name="radar_object_clustering_param_path" default="$(find-pkg-share radar_object_clustering)/config/radar_object_clustering.param.yaml"/>
```

Comment:
[Q] Is it intended to delete this `radar_object_clustering_param_path`?

Reply:
Thank you for checking.
Yes, I change to align to other launcher arg like [this PR](https://github.com/autowarefoundation/autoware.universe/pull/6158).